server/index.js

```diff
@@ -16,13 +18,25 @@ dotenv.config({ path: path.resolve(__dirname, '../.env') });
 
 const app = express();
 
+app.use(express.static(path.resolve(__dirname, '../public')));
+
 app.use(express.json());
 app.use('/api', games, makers);
 
 app.get('/*', (req, res) => {
-  const markup = ReactDOMServer.renderToString(<App />);
+  const context = {};
+
+  const markup = ReactDOMServer.renderToString(
+    <StaticRouter location={req.url} context={context}>
+      <App />
+    </StaticRouter>,
+  );
```

Comment:
Define the routes in ExpressJS
For each of those route's, you'll have a call to `ReactDOMServer.renderToString()`